app/dashboard/management/commands/activity_report.py

```diff
@@ -115,7 +115,7 @@ def format_tip(self, tip):
             'amount': tip.get_natural_value() * 10**18,
             'denomination': tip.tokenName,
             'amount_eth': tip.value_in_eth,
-            'amount_usdt': tip.value_in_usdt,
+            'amount_usdt': tip.value_in_usdt_now,
```

Comment:
> value_in_usdt_now

this is wrong.. for this report we actually want `value_in_usdt`

Reply:
Done, but I had to add value_in_usdt property to Tips. Since web3_created is not available there, I based on received_on timestamp.